org-roam-db.el

```diff
@@ -121,7 +121,7 @@ SQL can be either the emacsql vector representation, or a string."
   '((files
      [(file :unique :primary-key)
       (hash :not-null)
-      (last-modified :not-null)])
+      (meta :not-null)])
```

Comment:
Bump the db version

Reply:
Done in acaa3b4cd7c764c1c2dff71dca63e96fe2dbf5e4.